R/shims.R

```diff
@@ -497,7 +497,7 @@ require <- function(package,
 
 package_name <- function(package, character.only = FALSE) {
   if (!character.only) {
-    package <- as.character(rlang::quo_expr(package))
+    package <- as.character(rlang::quo_squash(package))
```

Comment:
What does `quo_squash()` do??

Reply:
`quo_squash()` is a replacement for `quo_expr()`. I believe it's the exact same function just renamed. According to the documentation "`quo_squash()` flattens all nested quosures within an expression"